index.html

```diff
@@ -6,11 +6,211 @@
+                    </br> from Cupcakes, Celebration Cakes,
+                    </br> Birthday Cakes, Chocolate cakes, Novelty cakes, Bespoke cakes
+                    </br> and even same-day Wedding Cakes.
+                    <br/><br/>If you can dream of it we can make it.<br/><br/>
+                    
+                    Hover over the image to the left and the images below for more information
+                </p>
+
+            </section>
+
+        <!-- NOTE: In order for this to work properly, the hierarchy and placement of these classes matters!! -->
+        <section class="cake-box">
+
+                <section class="hover-content"> 
+                        <div class="hover-content-overlay"></div>
+                        <img src="img/cake-cherriesHW3.jpg" class="maincake" class="hover-content-image" alt="A cake topped with cherries">
+                
+                        <div class="hover-content-details fadeIn-bottom">
+                            <h3 class="hover-content-title">Cakes</h3>
+                            <p>From Chocolate Cakes, to Fruity Cakes, to cheesecakes and so many more, there is a cake for any celebration and occasion.
+                                Why not personalise a special birthday message on your chosen cake to make the celebration extra special.
+                               ASTERISK ET OBELIX online cake range from classic cakes to the more showstopping cakes,
+                                there really is something for everyone.
+                            </p>
+                        </div>
+                </section>
+        </section>
+
+        <section class="FourCakes-FlexBox ">
+
+                <section class="ColumnFlexBox">
+
+                    <section class="RowFlexBox">
+
+                         <figure class="hoverwrap">
+                                <img id="close" src="img/cheesecakeHW3.jpg" class="display-a-cake" alt="A Cheese Cake">
+                                <!-- THE HOVER TEXT: -->
+                                <div class="hovercap">Our collection includes cakes that also make for delectable desserts, such as our elegant,
+                                                   creamy Strawberry Summer Cheesecake and Apricot Cheesecake
+                                </div>
+                        </figure>
+
+                        <figure class="hoverwrap">
+                            <img src="img/chocolate-cakeHW3.png" class="display-a-cake" alt="A Chocolate Cake">
+                            <!-- THE HOVER TEXT: -->
+                            <div class="hovercap">We are a little health-conscious and like to make sure our cakes are irresistibly delicious
+                                                   without using too much fat and sugar "just as much as needed to make something delicious". 
+                            </div>
+                        </figure>   
+                    </section>
```

Comment:
You do not need to group your images 2 by 2 here. Put them all in the same container, and then use CSS Grid to align the items 2 by 2 on mobile, and then all in a row on larger screens. It's be much cleaner and neater HTML, with just one parent and 4 child items, and your CSS should be more straight forward as well.